Polygon_mesh_processing/test/Polygon_mesh_processing/connected_component_polyhedron.cpp

```diff
@@ -71,17 +71,17 @@ void mesh_no_id(const char* argv1, const bool save_output)
   PMP::connected_component(fd, sm, std::back_inserter(cc));
 
   std::cerr << cc.size() << " faces in the CC of " << &*fd << std::endl;
-  boost::property_map<Mesh,boost::vertex_external_index_t>::type vim 
+  boost::property_map<Mesh,boost::vertex_external_index_t>::type vim
     = get(boost::vertex_external_index,sm);
 
-  boost::property_map<Mesh,boost::face_external_index_t>::type fim 
+  boost::property_map<Mesh,boost::face_external_index_t>::type fim
```

Comment:
this will trigger a unused variable warning

Reply:
How so ? It did not before, I only changed a whitespace.

Reply:
the map is no longer used AFAIS